### PR TITLE
report java crash from junit.py

### PR DESCRIPTION
**Pull request by renat-nosto**
# Pull Request Checklist

* [ ] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [ ] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [ ] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [ ] Have you updated the documentation?
* [ ] Have you added tests for any changed functionality?

# Helpful things

## Fixes

Fixes #xxxx

## Purpose

Logs explicitly the return code for java process in order to be able to notice the crashes 

## Background Context

Issues on the CI most likely due to OOM killer

## References

Are there any relevant issues / PRs / mailing lists discussions?

### CODE REVIEW

File: framework/pym/play/commands/junit.py [hunk 1]
```diff
@@ -38,16 +39,13 @@ def handle_sigint(signum, frame):
             process.kill()
 
 def execute(**kargs):
-    command = kargs.get("command")
     app = kargs.get("app")
     args = kargs.get("args")
-    env = kargs.get("env")
-    cmdloader = kargs.get("cmdloader")
-
```
Comment (renat-nosto) [inline framework/pym/play/commands/junit.py:46]: cleanups, removed ones were not used

File: framework/pym/play/commands/junit.py [hunk 2]
```diff
@@ -70,25 +68,19 @@ def junit(app, args):
...
     args.append('-Dplay.autotest')
     java_cmd = app.java_cmd(args, className='play.test.Runner')
     try:
-        process = subprocess.Popen (java_cmd, env=os.environ)
+        process = subprocess.Popen(java_cmd, env=os.environ)
         signal.signal(signal.SIGTERM, handle_sigterm)
         signal.signal(signal.SIGINT, handle_sigint)
         return_code = process.wait()
         if 0 != return_code:
+            print("~ java process exited with exit code %d" % return_code)
```
Comment (renat-nosto) [inline framework/pym/play/commands/junit.py:81]: Important bit, logging return code to be picked up by CI

File: framework/pym/play/commands/junit.py [hunk 3]
```diff
@@ -70,25 +68,19 @@ def junit(app, args):
...
         sys.exit(-1)
 
-    if os.path.exists(os.path.join(app.path, 'test-result/result.passed')):
-        print("~ All tests passed")
-        print("~")
-        testspassed = True
-    if os.path.exists(os.path.join(app.path, 'test-result/result.failed')):
-        print("~ Some tests have failed. See file://%s for results" % test_result)
-        print("~")
-        sys.exit(1)
```
Comment (renat-nosto) [inline framework/pym/play/commands/junit.py:94]: this particular files are not generated by `play.test.Runner`

Comment (leandroutn) [reply framework/pym/play/commands/junit.py:94]: I'm not sure what this means. Isn't it needed?

File: framework/pym/play/commands/junit.py (hunk 2 above)
Comment (leandroutn) [reply framework/pym/play/commands/junit.py:81]: very minor: "java process exited with code %"?

File: framework/pym/play/commands/junit.py [hunk 4]
```diff
@@ -38,16 +39,13 @@ def handle_sigint(signum, frame):
...
     app = kargs.get("app")
     args = kargs.get("args")
-    env = kargs.get("env")
-    cmdloader = kargs.get("cmdloader")
-
     junit(app, args)
 
 def junit(app, args):
     app.check()
+    print("~ Running junit")
```
Comment (leandroutn) [inline framework/pym/play/commands/junit.py:48]: is this needed?

File: framework/pym/play/commands/junit.py (hunk 3 above)
Comment (renat-nosto) [reply framework/pym/play/commands/junit.py:94]: I was confused by this code and the fact that neither `test-result/result.passed` nor `test-result/result.failed` is generated so i decided to clean it up, it doesn't look like these log messages are picked up by CI and there are another log messages about the results of the run from the java code

File: framework/pym/play/commands/junit.py (hunk 4 above)
Comment (renat-nosto) [reply framework/pym/play/commands/junit.py:48]: Not really, it rather was confirmation of what's executed on CI to tell apart from https://github.com/Nosto/play1/blob/master/framework/pym/play/commands/test.py#L25 , i don't mind cleaning this up

**Review by leandroutn — APPROVED** [review]
looks good to me 👍🏼

**Pull request merged**